File: Back/src/ProEventos.Domain/models/Palestrante.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.ConstrainedExecution;
using System.Threading.Tasks;

namespace ProEventos.Domain.models
{
    public class Palestrante
    {
        public int Id { get; set; }
        public string? Nome { get; set; }
        public string? MiniCurriculo { get; set; }
        public string? ImagemURL { get; set; }
        public string? Telefone { get; set; }
        public string? Email { get; set; }
        public IEnumerable<RedeSocial>? RedeSociais { get; set; }
        public IEnumerable<PalestranteEvento>? PalestrantesEventos { get; set; }

    }
}

File: Front/ProEvento-App/src/app/models/Palestrante.ts
import { PalestranteEvento } from "./PalestranteEvento";
import { RedeSocial } from "./RedeSocial";

export interface Palestrante {
     id: number;
     nome?: string;
     miniCurriculo?: string;
     imagemURL?: string;
     telefone?: string;
     email?: string;
     redeSociais?: RedeSocial[];
     palestrantesEventos: PalestranteEvento[];
}

